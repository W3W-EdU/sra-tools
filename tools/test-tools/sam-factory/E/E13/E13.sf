#bam-load error test #13 : a pair of alignments has a secondary alignment - but the length differ

r:type=random,name=R1,length=5000

# write all references into one file as FASTA
ref-out:E13.fasta

# write the final SAM-output into this file
sam-out:E13.sam

# write a config-file for bam-load
config:E13.config

p:name=A1,pos=100,cigar=30M
p:name=A1,pos=300,cigar=20M
s:name=A1,pos=200,cigar=25M

#some additional ( correct ) alignments
p:name=B,repeat=50
p:name=B,repeat=50
