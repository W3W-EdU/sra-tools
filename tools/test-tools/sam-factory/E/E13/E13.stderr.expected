warn: libs/vdb/phys-cmn.c:733:VPhysicalProdColumnIdRange: constraint violated while reading file within application support module - Secondary alignment for 'A1' has different length (25) than previously recorded primary alignment; discarding secondary alignment.
